QueryLib.js

```diff
@@ -0,0 +1,60 @@
+define([], function() {
```

Comment:
This is an AMD module. It uses `define`. It is not supposed to be directly imported from a HTML file but can be imported from another AMD module or from the _"main"_ module (the one that uses `require`).